test/dynamo/test_misc.py

```diff
@@ -3260,6 +3260,21 @@ def fn(x, y):
         res = opt_fn(x, y)
         self.assertTrue(same(ref, res))
 
+    def test_nonexistent_tensor_attr(self):
+        def fn(x):
+            try:
+                unsupported = x.nonexistent_tensor_attr
+                x = torch.nn.utils.rnn.PackedSequence(x, unsupported)
```

Comment:
Isn't this line dead? Shouldn't we immediately have thrown an error on the attribute access?